app/alembic/env.py

```diff
@@ -1,12 +1,11 @@
 from __future__ import with_statement
 
-import os
 from logging.config import fileConfig
 
-from sqlalchemy import engine_from_config, pool
-
+import decouple
 from alembic import context
 from app.db.base import Base  # noqa
+from sqlalchemy import engine_from_config, pool
```

Comment:
what is this for?

Reply:
Check the whole diff, that's just imports getting rearranged

Reply:
oh